blockchain/blockchain_test.go

```diff
@@ -1,224 +0,0 @@
-// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-package blockchain
-
-import (
-	"context"
-	"fmt"
-	"math/big"
-	"testing"
-	"time"
-
-	"github.com/ava-labs/avalanche-tooling-sdk-go/keychain"
-	"github.com/ava-labs/avalanche-tooling-sdk-go/network"
-	"github.com/ava-labs/avalanche-tooling-sdk-go/vm"
-	"github.com/ava-labs/avalanche-tooling-sdk-go/wallet"
-	"github.com/ava-labs/avalanchego/ids"
-	"github.com/ava-labs/avalanchego/utils/formatting/address"
-	"github.com/ava-labs/avalanchego/wallet/subnet/primary"
-	"github.com/ava-labs/libevm/common"
-	"github.com/ava-labs/libevm/core"
-	"github.com/ava-labs/subnet-evm/params/extras"
-	"github.com/ava-labs/subnet-evm/utils"
-
-	"github.com/stretchr/testify/require"
-)
-
-func getDefaultSubnetEVMGenesis() SubnetParams {
-	allocation := core.GenesisAlloc{}
-	defaultAmount, _ := new(big.Int).SetString(vm.DefaultEvmAirdropAmount, 10)
-	allocation[common.HexToAddress("INITIAL_ALLOCATION_ADDRESS")] = core.GenesisAccount{
-		Balance: defaultAmount,
-	}
-	genesisBlock0Timestamp := utils.TimeToNewUint64(time.Now())
-	return SubnetParams{
-		SubnetEVM: &SubnetEVMParams{
-			ChainID:     big.NewInt(123456),
-			FeeConfig:   vm.StarterFeeConfig,
-			Allocation:  allocation,
-			Precompiles: extras.Precompiles{},
-			Timestamp:   genesisBlock0Timestamp,
-		},
-		Name: "TestSubnet",
-	}
-}
-
-func TestSubnetDeploy(t *testing.T) {
-	require := require.New(t)
-	subnetParams := getDefaultSubnetEVMGenesis()
-	newSubnet, err := New(&subnetParams)
-	require.NoError(err)
-	network := network.FujiNetwork()
-
-	keychain, err := keychain.NewKeychain(network, "KEY_PATH", nil)
-	require.NoError(err)
-
-	controlKeys := keychain.Addresses().List()
-	subnetAuthKeys := keychain.Addresses().List()
-	threshold := 1
-	newSubnet.SetSubnetControlParams(controlKeys, uint32(threshold))
-	wallet, err := wallet.New(
-		context.Background(),
-		network.Endpoint,
-		keychain.Keychain,
-		primary.WalletConfig{
-			SubnetIDs: nil,
-		},
-	)
-	require.NoError(err)
-	deploySubnetTx, err := newSubnet.CreateSubnetTx(wallet)
-	require.NoError(err)
-	subnetID, err := newSubnet.Commit(*deploySubnetTx, wallet, true)
-	require.NoError(err)
-	fmt.Printf("subnetID %s \n", subnetID.String())
-	time.Sleep(2 * time.Second)
-	newSubnet.SetSubnetAuthKeys(subnetAuthKeys)
-	deployChainTx, err := newSubnet.CreateBlockchainTx(wallet)
-	require.NoError(err)
-	blockchainID, err := newSubnet.Commit(*deployChainTx, wallet, true)
-	require.NoError(err)
-	fmt.Printf("blockchainID %s \n", blockchainID.String())
-}
-
-func TestSubnetDeployMultiSig(t *testing.T) {
```

Comment:
From a quick glance here, it looks like tests were removed because the wallet package dependency was removed. And these tests specifically assume wallet was part of the blockchain package?